schema/2.0.0/seurat_encoding.md

```diff
@@ -28,7 +28,7 @@ The following sections describe the individual components of a dataset as encode
 
 ### Data matrix
 
-Matrix data is stored in the slot `assays` under the element `RNA`.
+Matrix data is stored in the slot `assays` under the element `RNA`. Seurat allows elements of `assasy` to have any arbitrary name, but all matrix data from cellxgene will be stored in `RNA`.
 
 #### Raw data
 
```

Comment:
The style is inconsistent between the raw and final sections. (Final includes a '.' at the end of the value sentence fragment).

Suggest:

<code>Matrix::dgCMatrix</code> object for sparse matrices
<code>base::matrix</code> object for dense matrices